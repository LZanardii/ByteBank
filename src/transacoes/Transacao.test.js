import { render } from "@testing-library/react";
import React from "react";
import Transacao from "./Transacao";


describe("SnapshotTest. O componente nÃ£o deve ser alterado", () => {
     
    it("Deve manter-se o mesmo snapshot", () => {
        const {container} = render(<Transacao
            data = "10/08/2022"
            tipo = "saque"
            valor = "25"
        />)

        expect(container).toMatchSnapshot()
    })  
})
