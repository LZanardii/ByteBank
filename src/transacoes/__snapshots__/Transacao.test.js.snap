// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest. O componente n√£o deve ser alterado Deve manter-se o mesmo snapshot 1`] = `
<div>
  <div
    class="transacao-container"
    data-testid="transacaoContainer"
  >
    <p>
      10/08/2022
    </p>
    <p>
      saque
    </p>
    <p>
      R$ 
      25
    </p>
  </div>
</div>
`;
